code/game/turfs/turf.dm

```diff
@@ -75,6 +75,9 @@
 
 	var/holomap_draw_override = HOLOMAP_DRAW_NORMAL
 
+	var/list/border_objects //Only exists when there are border objects on the turf.
+	var/atom/movable/bump_target //If this var is not null, bumping this turf bumps this object instead. Used for border objects. Unset immediately when bumped.
```

Comment:
I could have sworn I tested this shit and the relate garbage to make border objects get bumped first, but apparently not! It doesn't work. Yes there's code I didn't push that actually implements the `bump_target` thing. Turns out the turf is, for some fucking reason, the *last* thing you bump, not the first! So there is, in fact, no way to ensure border objects get bumped first without reimplementing all of BYOND's collision code again! This PR is on hold until Lummox responds to http://www.byond.com/forum/post/2655423 because much of it has to just be reverted if no such control is added.